packages/admin/docs/01-installation.md

```diff
@@ -61,6 +61,8 @@ If you wish to translate the package, you may publish the language files using:
 
 ```bash
 php artisan vendor:publish --tag=filament-translations
+php artisan vendor:publish --tag=filament-tables-translations
```

Comment:
If we're going to have all packages listed separately here, let's have forms before tables (sorted like elsewhere).